sample/src/main/java/com/cloudinary/android/sample/app/Utils.java

```diff
@@ -94,11 +95,16 @@ public static Uri toUri(Context context, int resourceId) {
                 + '/' + res.getResourceTypeName(resourceId) + '/' + res.getResourceEntryName(resourceId));
     }
 
+    @SuppressWarnings("deprecation")
     public static int getScreenWidth(Context context) {
         WindowManager window = (WindowManager) context.getSystemService(Context.WINDOW_SERVICE);
         Point point = new Point();
-        window.getDefaultDisplay().getSize(point);
-        return point.x;
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.R) {
+            return window.getCurrentWindowMetrics().getBounds().width();
+        } else {
```

Comment:
no need for `else`

Reply:
The else is for Old android version < Android R